cmd/kubectl-gadget/bcck8s.go

```diff
@@ -212,7 +212,7 @@ func bccCmd(subCommand, bccScript string) func(*cobra.Command, []string) {
 
 		namespaceFilter := ""
 		if !params.AllNamespaces {
-			namespace := utils.GetNamespace()
+			namespace := utils.GetNamespace(false)
```

Comment:
IIUC you're trying to avoid setting the namespace when the user doesn't specify the `-n` flag but has a default namespace configured in the kubeconfig to avoid an error that indicates that filtering is not supported by the biolatency gadget. It makes sense, however I think we could implement it in another way. 

First of all, take into consideration https://github.com/kinvolk/inspektor-gadget/pull/378 that will change a bit how we're handling the namespace around. There's now a `Namespace` field in `CommonFlags`. 

I think `CommonFlags` could be extended with a `NamespaceOverridden` field, then different gadgets could understand if the namespace was overridden or not. I wouldn't create the `IgnoreKubeConfigNamespace` field in `TraceConfig`, I'd rather handle this case in the biolatency logic setting `params.Namespace = ""` before creating theTraceConfig. 

```
if params.Namespace != "" && !params.NamespaceOverridden {
  return error ( "you can't use -n with biolatency")
}

params.Namespace = "" // Just in case it was set from the kubeconfig. 

// create traceconfig as usual...
```

Reply:
Yeah, you understood correctly.

Sorry I wasn't aware of the changes being introduced by #378.

So the idea is to set `params.NamespaceOverridden` to `true` if the default namespace configured in the kubeconfig was stored in `params.Namespace`, correct? I think that's a more clean solution!

Reply:
>  correct? 

yes!

Reply:
Done

Reply:
I just don't like the fact that right now the `stop` and `list` commands will still accept the `--namespace` flag but they will just ignore it.